src/Controls/src/Core/Compatibility/Handlers/Shell/Android/ShellToolbarTracker.cs

```diff
@@ -185,7 +185,10 @@ protected override void Dispose(bool disposing)
 				_currentMenuItems?.Clear();
 				_currentToolbarItems?.Clear();
 
+				_drawerLayout.RemoveDrawerListener(_drawerToggle);
```

Comment:
Fix content pages not Garbage collected
https://github.com/xamarin/Xamarin.Forms/pull/14717